src/renderer/config-blocks/Comment.svelte

```diff
@@ -1,17 +1,16 @@
-<script context="module">
+<script lang="ts" context="module">
+  import type { ActionBlockInformation } from "./ActionBlockInformation.ts";
   // Component for the untoggled "header" of the component
   import RegularActionBlockFace from "./headers/RegularActionBlockFace.svelte";
   export const header = RegularActionBlockFace;
 
   // config descriptor parameters
-  export const information = {
+  export const information: ActionBlockInformation = {
     short: "c",
     name: "Comment",
     rendering: "standard",
-    toggleable: false,
```

Comment:
why is toggleable attribute removed? Comment block should always render opened, ready for editing!

Reply:
The property toggleable was present in ElementNames.svelte twice with different values. The unnecessary and incorrect one (above) was removed.

The toggleable property is currently used for defining that whether an action block has an openable body or not. Setting toggleable to false causes the action block to always "show it's face", e.g.: If block, For loop block.

An always open ElementName and CodeBlock is doable, similiarly to the If block and For loop.

Reply:
Eventtype is removed since the behaviour of the filtering of unaddable action blocks is defined in ActionPicker.svelte.
The filtering is using a variable (eventString) that defines the type of the event, and the config blocks that are unaddable on that event are manually filtered out.